### PR TITLE
Feature/new timeline chart

### PR DESCRIPTION
**Pull request by urchykli**

### CODE REVIEW

**Comment by urchykli** [comment]
Changes:
- Created Google sheet that parses the data and will update automatically
- Add link to sheet
- Change title font
- Remove subtitle on export
- Change credits to Missile Threat url on export

File: hc-bar/js/bar.js [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+  googleSpreadsheetKey: "1AKM59m3iaOSSQgVsIoDcSWxjcpX781JIuBwB9YdDgFs",
+  googleSpreadsheetWorksheet: 1,
+  switchRowsAndColumns: true,
+  parsed: function parsed(columns) {
+    // set default values
+    var drilldown = "";
+    var dataObject = { other: {}, strike: {}, intercept: {} };
+    var dataArray = [];
+    var drilldownObject = {};
+    var event = "";
```
Comment (smaraghi) [inline hc-bar/js/bar.js]: I believe that this event object is never used in the code. Please remove it as its value is never read.

File: hc-bar/js/bar.js [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+      if (interceptColored) {
+        eventRow = "intercept";
+      } else if (strikeColored) {
+        eventRow = "strike";
+      } else {
+        eventRow = "other";
+      }
+      // Remove links from the text that will appear in the tooltip
+      var eventText = row[10].substr(0, row[10].indexOf("<a "));
+      // var eventText = row[10]
```
Comment (smaraghi) [inline hc-bar/js/bar.js]: Lets remove this code comment (var eventText...)

**Review by smaraghi — CHANGES_REQUESTED** [review]
Thank you for all of your hard work!!

Please remove unnecessary code comments as well as the object referenced below.

Thanks!

**Comment by urchykli** [comment]
Removed unused event variable and unnecessary code comments

**Review by smaraghi — APPROVED** [review]

**Pull request merged**